src/main/java/it/gov/pagopa/pu/workflow/wf/classification/iuf/classification/IufClassificationWFImpl.java

```diff
@@ -86,15 +87,20 @@ public void notifyPaymentsReporting(IufClassificationNotifyPaymentsReportingSign
 
     log.info("Handling payments reporting notification in iuf classification: {}", signalDTO);
     Long clearedResult = clearClassifyIufActivity.deleteClassificationByIuf(
-      signalDTO.getOrganizationId(),
+      signalDTO.getPaymentsReportingTransferDTO().getOrgId(),
       signalDTO.getIuf());
 
     log.info("IUF receipt classification cleared cleared {} records for {}", clearedResult, signalDTO);
+    Transfer2ClassifyDTO transfer2ClassifyDTO = Transfer2ClassifyDTO.builder()
+      .iur(signalDTO.getPaymentsReportingTransferDTO().getIur())
+      .iuv(signalDTO.getPaymentsReportingTransferDTO().getIuv())
+      .transferIndex(signalDTO.getPaymentsReportingTransferDTO().getTransferIndex())
+      .build();
 
     toNotify.add(IufClassificationActivityResult.builder()
-      .organizationId(signalDTO.getOrganizationId())
+      .organizationId(signalDTO.getPaymentsReportingTransferDTO().getOrgId())
       .success(true)
-      .transfers2classify(signalDTO.getTransfers2classify())
+      .transfers2classify(List.of(transfer2ClassifyDTO))
```

Comment:
Why List.of? the paymentsReporting ingestion is supposed to return a list of transfer! not just one!